third_party/BUILD

```diff
@@ -469,9 +469,9 @@ UNNECESSARY_DYNAMIC_LIBRARIES = select({
     "//src/conditions:arm": "*.so *.jnilib *.dll",
     "//src/conditions:linux_aarch64": "*.so *.jnilib *.dll",
     "//src/conditions:linux_ppc": "*.so *.jnilib *.dll",
+    "//src/conditions:linux_s390x": "*.so *.jnilib *.dll",
     # Play it safe -- better have a big binary than a slow binary
-    # zip -d does require an argument. Supply something bogus.
-    "//conditions:default": "*.bogusextension",
+    "//conditions:default": "",
```

Comment:
This fulfills point #2, right?

Reply:
Correct.  The empty string means nothing is removed.  The test for the empty string in the shell execution keeps unzip from being run as a no-op (causing a failure).

Reply:
Thanks for clarifying!